server/tests/admin.js

```diff
@@ -49,4 +49,40 @@ describe('ADMIN CONTROLLER', () => {
         });
     });
   });
+  describe('PUT /api/v1/requests/:id/approve', () => {
+    it('should allow an authenticated admin to update a request', (done) => {
+      chai.request(app)
+        .put('/api/v1/requests/1/approve')
+        .set('token', userToken)
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body).to.be.an('object');
+          expect(res.body.message).to.equal('request approved successfully');
+          expect(res.body.status).to.be.equal('success');
```

Comment:
Test other properties that can be tested.

Reply:
okay, did that as well